src/ddmd/cond.d

```diff
@@ -65,6 +65,223 @@ extern (C++) abstract class Condition : RootObject
 
 /***********************************************************
  */
+import ddmd.statement: ForeachStatement, ForeachRangeStatement, ReturnStatement, ExpStatement, CompoundStatement, Statement;
+import ddmd.declaration: VarDeclaration, StructDeclaration, STCstatic;
+import ddmd.func: FuncDeclaration, FuncLiteralDeclaration;
+import ddmd.aliasthis: AliasThis;
+import ddmd.dtemplate: TemplateTupleParameter, TemplateDeclaration;
```

Comment:
Imports should be at top of file. Selective imports should be only at scope level.